_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
   - imnotashrimp
 ---
 
-To create a new alert, you'll start in Kibana so you can test the query you want to use. After that, you'll continue to the Create a New Alert page, where you can configure the alert and notification settings.
+When you set out to create an alert,
+you'll need to start with a query.
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
+
+Otherwise, if you already have a query and filters,
+you can create a blank alert
+from the top menu (**Alerts & Events > New alert**)
+or from the _Alert definitions_ page
 
 Community plans have a maximum of 50 alerts.
 {:.info-box.note}
 
-If you want help updating an existing alert, you can [skip the first part of this page](#to-configure-an-alert).
-
-#### To start a new alert
+#### To create an alert in Kibana
 
-![Kibana query bar]({{site.baseurl}}/images/kibana/kibana--query-bar.png)
+![Kibana 7 search bar]({{site.baseurl}}/images/kibana/kibana-7-search-bar.png)
 
-##### Set your query in Kibana
+Refine your search query and filters,
+and review the returned logs
+to make sure you got the expected results.
```

Comment:
We have a char limit for the search box

Reply:
I'll keep an ear out if users are hitting this limit.